addons/registry-route/registry-route.addon

```diff
@@ -0,0 +1,18 @@
+# Name: registry-route
+# Description: Create an edge terminated route for the OpenShift registry
+
+openshift admin ca create-server-cert --signer-cert=/var/lib/origin/openshift.local.config/master/ca.crt --signer-key=/var/lib/origin/openshift.local.config/master/ca.key --signer-serial=/var/lib/origin/openshift.local.config/master/ca.serial.txt --hostnames='docker-registry-default.#{routing-suffix},docker-registry-default.#{routing-suffix}:443,docker-registry.default.svc.cluster.local,172.30.1.1' --cert=/etc/secrets/registry.crt --key=/etc/secrets/registry.key
+openshift cli create route edge docker-registry --service=docker-registry --ca-cert=/var/lib/origin/openshift.local.config/master/ca.crt --cert=/etc/secrets/registry.crt --key=/etc/secrets/registry.key --hostname='docker-registry-default.#{routing-suffix}'
+
+ssh sudo mkdir -p /etc/docker/certs.d/docker-registry-default.#{routing-suffix}:443
+ssh sudo chown docker /etc/docker/certs.d/docker-registry-default.#{routing-suffix}:443
+
+ssh sudo mkdir -p /etc/docker/certs.d/docker-registry-default.#{routing-suffix}
+ssh sudo chown docker /etc/docker/certs.d/docker-registry-default.#{routing-suffix}
+
+docker cp origin:/var/lib/origin/openshift.local.config/master/ca.crt /etc/docker/certs.d/docker-registry-default.#{routing-suffix}:443/ca.crt
+docker cp origin:/var/lib/origin/openshift.local.config/master/ca.crt /etc/docker/certs.d/docker-registry-default.#{routing-suffix}/ca.crt
+
+echo Add-on '#{addon-name}' created docker-registry route. You can now login to the OpenShift registry via:
```

Comment:
Suggestion  /`You can now login to the OpenShift registry via:`/Please run following commands to login to the OpenShift docker registry/